### PR TITLE
[WIP] Anonymise and setup user

### PR DESCRIPTION
**Pull request by astroash**
relates #4

### CODE REVIEW

**Comment by ameliejyc** [comment]
Oh hello. This is unrelated but given you're doing stuff, can we add in `eslint-config-prettier` to use prettier's formatting rather than eslint's craziness?

Also my prettier config is no trailing commas and single quotes.

**Comment by astroash** [comment]
we could write our own eslint config? Or pick something less opinionated than airbnb? I've messed around with my prettier settings loads, it seems silly for us both to manually set them the same.

**Comment by ameliejyc** [comment]
We don't even have prettier installed on this project actually, lol. 

Hmm we can do eslint config if you like, tbh I like airbnb's opinionatedness tho.

**Comment by astroash** [comment]
Can you install prettier? I _think_ it’s an editor thing and it just uses your eslint config file. If we’re both using vs code we could put our prettier setting in the workstation config and it will overwrite our user settings.

**Comment by astroash** [comment]
This PR could start to get quite big (and I'm making some structural changes). Feel free to merge if you wanna work on stuff.

**Comment by ameliejyc** [comment]
It's not just an editor thing - you can install prettier to the project and set the config. We should just sort it out in person when we next see each other! I might take a look at this pr this weekend but tbh pretty busy so no promises :(

File: server/database/db_build.sql [hunk 1]
```diff
@@ -4,7 +4,7 @@ DROP TABLE IF EXISTS users, issues, comments CASCADE;
 
 CREATE TABLE users (
   id            SERIAL         PRIMARY KEY,
-  username      VARCHAR(300)   NOT NULL,
+  username      VARCHAR(64)   NOT NULL,
```
Comment (ameliejyc) [inline server/database/db_build.sql]: What's up with this alignment?

File: package.json [hunk 2]
```diff
@@ -21,12 +21,13 @@
...
   "dependencies": {
     "body-parser": "^1.18.2",
+    "cookie-session": "^2.0.0-beta.3",
     "env2": "^2.2.0",
     "express": "^4.16.2",
     "morgan": "^1.9.0",
     "passport": "^0.4.0",
     "passport-github": "^1.1.0",
-    "pg": "^7.4.0"
+    "pg-promise": "^7.3.3"
```
Comment (ameliejyc) [inline package.json:30]: I heard from a good source that this is incorporated in pg, rendering it absolutely unnecessary.

Comment (ameliejyc) [reply package.json:30]: I heard from a good source that actually we should just leave it and it's okay.

**Review by ameliejyc — APPROVED** [review]

**Pull request merged**